libs/experimental/langchain_experimental/autonomous_agents/autogpt/agent.py

```diff
@@ -133,11 +133,11 @@ def run(self, goals: List[str]) -> str:
                 f"Assistant Reply: {assistant_reply} " f"\nResult: {result} "
             )
             if self.feedback_tool is not None:
-                feedback = f"\n{self.feedback_tool.run('Input: ')}"
+                feedback = f"{self.feedback_tool.run('Input: ')}"
```

Comment:
I kept the f-string in case there's a reason to make sure this run output is a string, but I can remove it altogether if preferred.